store/products.ts

```diff
@@ -40,18 +40,21 @@ const productsModule: Module<ProductsState, any> = {
       const products = await dispatch('fetch', {
         placements: 'home_page.2020_start_few'
       })
-      commit('gotProducts', products)
+      commit('gotProductsFew', products)
     },
     async loadFull({ state, dispatch, commit }) {
       if (state.didShowMore) {
         event('full-exists')
         return
       }
+      if (process.env.NODE_ENV === 'development') {
+        await wait(3000)
+      }
       event('load-full')
       const products = await dispatch('fetch', {
         placements: 'home_page.2020_start_full'
       })
-      commit('gotProducts', products)
+      commit('gotProductsFull', products)
```

Comment:
You could also just add another mutation called `inited` instead of duplicating all that code. Something like this:

`commit('inited')`

and then 

```js
inited(state: ProductsState) {
  state.inited = true
},
```